packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -59,6 +59,7 @@ const propTypes = {
 
   /**
    * Child element to be displayed on the right end of the header.
+   * This is intended to be used with the CollapsibleMenuView.
    */
   children: PropTypes.element,
```

Comment:
children implies that I can pass in an array of elements, this is allowing me to only pass in 1, should we allow this to be of type node, else this this name needs to be changed to child

Reply:
Children is actually a keyword, so the name needs to remain the same. I don't mind changing it to node, but in the war room we discussed leaving the prop value at element until someone requests the functionality to pass additional elements.